pipeline/backend/common/script_posix.go

```diff
@@ -53,6 +53,7 @@ fi
 unset CI_NETRC_USERNAME
 unset CI_NETRC_PASSWORD
 unset CI_SCRIPT
+chown -R $(whoami) /woodpecker
```

Comment:
that will work if you are root but it will fail for user contaierns that want to alter root owned files ...
... well they did now work properly before either but now the script will fail for sure

Reply:
Thanks. I am just playing around, hence WIP label. I will ask actively if feedback is needed :)